dd-sdk-android-core/src/testFixtures/kotlin/com/datadog/android/api/instrumentation/network/RequestInfoAssert.kt

```diff
@@ -0,0 +1,38 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.api.instrumentation.network
+
+import org.assertj.core.api.AbstractObjectAssert
+import org.assertj.core.api.Assertions
```

Comment:
minor: will allow to remove `Assertions` prefix

```suggestion
import org.assertj.core.api.Assertions.assertThat
```